docset/winserver2022-ps/failoverclusters/Set-ClusterStorageSpacesDirectDisk.md

```diff
@@ -34,14 +34,24 @@ S2D. Alternatively, you can run this cmdlet after you enable S2D.
 
 ### Example 1: Configure disks not to be claimed
 
-```
-PS C:\> Set-ClusterStorageSpacesDirectDisk -CimSession "K0619-C1.contoso.com" -CanBeClaimed:$False -PhysicalDiskIds "55CD2E404B75A3FC","50014EE05950DD7C"
+```powershell
+$parameters = @{
+    CimSession = 'K0619-C1.contoso.com'
+    CanBeClaimed = $False
+    PhysicalDiskIds = '55CD2E404B75A3FC',
+                      '50014EE05950DD7C'
```

Comment:
I'd recommend either using `@()` to group these entries or have them on a single line if that fits.

```suggestion
    PhysicalDiskIds = @(
        '55CD2E404B75A3FC'
        '50014EE05950DD7C'
    )
```

```suggestion
    PhysicalDiskIds = '55CD2E404B75A3FC', '50014EE05950DD7C'
```